msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -209,6 +213,19 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office app for Android
+
+To view your app running in Office app for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
```

Comment:
Could we append this sentence to the 1st step: If the Office app for Android was already running prior to sideloading your app in Teams, you'll need to restart it in order to see it among your installed apps.